app/src/store/voice/call/outgoingCall.ts

```diff
@@ -49,9 +49,11 @@ export const makeOutgoingCall = createAsyncThunk<
       outgoingCall.on(TwilioCall.Event.Reconnecting, (error) =>
         console.error('Reconnecting:', error),
```

Comment:
Should we add `if (error)` to these cases as well?

Reply:
These events should only ever be emitted when there are errors. Meaning that those errors are never `undefined` and are always truthy. The only exception is the `Disconnected` event where the listener param is `error?` since a disconnect can occur with or without an error.